tasks_downstream.py

```diff
@@ -428,7 +428,7 @@ def git_aggregate(c):
     """
     with c.cd(str(PROJECT_ROOT)):
         c.run(
-            "docker-compose --file setup-devel.yaml run --rm odoo",
+            "docker-compose --file setup-devel.yaml run --rm -T odoo",
             env=UID_ENV,
             pty=True,
```

Comment:
I understand that then it is not necessary to launch it with "pty" either.... no?

Reply:
Correct.